ls1intum/Artemis
# `Development`: Add documentation for the Cypress Bamboo build plans

### Description

In this PR I added documentation for the Bamboo buildplans executing the Cypress test suite.

### Steps for testing

1. [Build the Artemis documentation locally](https://github.com/ls1intum/Artemis/blob/documentation/cypress-deployment/docs/README.md)
2. Open the documentation html. You should see a new section in the Contributor Guide for the cypress bamboo build plans.
3. Check the cypress build plan documentation for typos and if it is understandable


### Review Progress





#### Documentation Review

#### Screenshots
The deployment diagram for the Cypress Github Integration
![cypress_bamboo_deployment_diagram](https://user-images.githubusercontent.com/12861668/154942707-cf7b5438-5ec9-4349-b925-fb56993c25bc.svg)

The deployment diagram for the Artemis deployment on test server 3
![cypress_test_environment_deployment_diagram](https://user-images.githubusercontent.com/12861668/154998298-8e936065-1ade-47ac-8d18-7aa3585d4b42.svg)
